src/ui/components/shared/LoadingScreen.tsx

```diff
@@ -36,15 +36,14 @@ export function LoadingScreenTemplate({
 
   return (
     <BubbleViewportWrapper>
-      <div className="relative flex w-96 flex-col items-center space-y-1 rounded-lg bg-loadingBoxes p-8 py-12 shadow-sm">
-        <div className="flex flex-col items-center space-y-8">
+      <div className="relative flex w-96 flex-col items-center space-y-1 rounded-lg bg-loadingBoxes p-4 py-12 shadow-sm">
+        <div className="flex flex-col items-center space-y-2">
           <div className="h-32 w-32" onClick={changeHoverboardColor}>
             <Hoverboard color={hoverboardColor} />
           </div>
           {children}
         </div>
       </div>
-      {showTips ? <LoadingTips /> : null}
```

Comment:
Can you also delete the no-longer-in-use `showTips` variable that appears 3 other places in this file?